zomes/blog/capabilities/main/manifest.json

```diff
@@ -0,0 +1,42 @@
+{
+  "capability": {
+    "membrane": "agent"
+  },
+
+  "fn_declarations": [
```

Comment:
I actually want to change this to: ```abi```

Reply:
I get why ABI (and also what they use in Ethereum).  Hmmm.  But note that we also have signals that need to be delcared.